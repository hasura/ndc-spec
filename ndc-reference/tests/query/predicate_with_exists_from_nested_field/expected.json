[
  {
    "rows": [
      {
        "name": "Chalmers University of Technology",
        "location": {
          "country_id": 2,
          "country": {
            "rows": [
              {
                "id": 2,
                "name": "Sweden",
                "area_km2": 450295
              }
            ]
          }
        }
      },
      {
        "name": "Stockholm University",
        "location": {
          "country_id": 2,
          "country": {
            "rows": [
              {
                "id": 2,
                "name": "Sweden",
                "area_km2": 450295
              }
            ]
          }
        }
      }
    ]
  }
]