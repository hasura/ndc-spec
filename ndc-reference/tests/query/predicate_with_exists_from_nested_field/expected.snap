---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/predicate_with_exists_from_nested_field/request.json
---
[
  {
    "rows": [
      {
        "name": "Chalmers University of Technology",
        "location": {
          "country_id": 2,
          "country": {
            "rows": [
              {
                "id": 2,
                "name": "Sweden",
                "area_km2": 450295
              }
            ]
          }
        }
      },
      {
        "name": "Stockholm University",
        "location": {
          "country_id": 2,
          "country": {
            "rows": [
              {
                "id": 2,
                "name": "Sweden",
                "area_km2": 450295
              }
            ]
          }
        }
      }
    ]
  }
]
